Doc/library/dis.rst

```diff
@@ -316,8 +316,19 @@ details of bytecode instructions as :class:`Instruction` instances:
 
       ``True`` if other code jumps to here, otherwise ``False``
 
+
+   .. data:: positions
+
+      Optional namedtuple ``(lineno, end_lineno, col_offset, end_col_offset)``
```

Comment:
I think we should explicitly mention the `Positions` class.

Reply:
Yes, I was thinking about it. So, here are my thoughts.

- If we mention it, we need to document it on the same page
- This page does not have a clear place for it, only below `Instruction` class under `Python Bytecode Instructions` header. It feels foreign to me
- On the other hand, if we don't mention exact class - we have more freedom in the future: it is not "really public" in CPython's rules
- `Positions` itself is not a real "class", it does not have any methods / interesting props / etc. It is just a structure with only 4 self-explanatory fields
- Documentation can be fixed later at any point if needed

Do you agree? 🙂

Reply:
`Optional namedtuple` does not sound really convincing, my 2cts is that we should document it. You can add the documentation right after the `Bytecode`, since it is practically the only use case.

Reply:
Done!